content/blog/qmstr-newsletter-6.md

```diff
@@ -0,0 +1,111 @@
++++
+title = "Quartermaster v0.1 development update #3: It works - now make it easier"
+date = "2018-03-13T00:00:00+01:00"
+tags = ["qmstr", "FOSS", "compliance", "LFOSLS"]
+categories = ["qmstr"]
+banner = "img/blog/qmstr-newsletter-6-title.jpg"
++++
+
+In sprint #3 towards Quartermaster v0.1, we made it easer to package
+the master into a Docker container using
+a
+[multi-stage Docker file](https://docs.docker.com/develop/develop-images/multistage-build/),
+merged the `qmstr-container` repository into the `qmstr` one, set up
+more CI to endure quality of the master HEAD and the incoming PRs,
+extended license detection with ScanCode and Ninka, and began
+implementing the reporting API endpoint. You can now see the build and
+test results both for incoming PRs as well as for the master mainline
+branch. Finally we prepared Quartermaster to be presented at
+the
+[Open Source Leadership Summit](https://events.linuxfoundation.org/events/open-source-leadership-summit-2018/). We
+made good progress, even though the setup is still a bit rough at the
+edges. It works, but the APIs are not yet as modular as we want them
+to be. We will focus on that in the next sprint.
+<!--more-->
+
+# Build, analyzers and reporters are there, but need to be refactored
+
+We finished implementing the modules that extend the build graph with
+author and license information using Ninka and ScanCode. The modules
+run in the _Analysis_ phase, which is triggered by `qmstr-cli`. They
+are, however, still performed in-process with the master. This
+violates our concept that the modules should not become derivative
+works of the master, so that module vendors are free to choose
+licenses for their modules that fit their needs. The analysis modules
+need to be refactored so that they run as separate processes executed
+by the master.
+
+{{< figure src="/img/blog/qmstr-newsletter-6-phases.png" alt="QMSTR Workflow Phases and Tasks" width="100%" >}}
+
+&nbsp;
+
+The architecture overview for Quartermaster illustrates the concepts
+of phases and functions that are key to the design. During the
+_Construction_ phase, the build graph and the necessary project
+metadata is aggregated in the data model. Once the build is complete,
```

Comment:
the build graph and the necessary project metadata is aggregated in the data model.

I believe it's plural --> "..metadata ARE aggregated.."